src/common/logger/CMakeLists.txt

```diff
@@ -24,9 +24,7 @@ endif()
 
 if(WIN32)
     set(SOURCES src/logger_win.cpp)
-elseif(APPLE)
-    set(SOURCES src/logger_macos.cpp)
-elseif(UNIX)
+else()
     set(SOURCES src/logger_linux.cpp)
```

Comment:
If Linux and macOS are using the same implementation, would it make sense to name this file `logger_unix.cpp`? Or something similar.